src/cbl-mariner/2.0/fpm/amd64/Dockerfile

```diff
@@ -1,5 +1,9 @@
 FROM mcr.microsoft.com/cbl-mariner/base/core:2.0
 RUN tdnf install -y \
         ruby \
+        # Provides useradd, needed by Azure DevOps
+        shadow-utils \
+        # Provides su, needed by Azure DevOps
+        util-linux \
```

Comment:
What exactly does `needed by Azure DevOps` mean?

Reply:
Hmm, I guess I thought the issue I was seeing in ASP.NET was because we explicitly call `useradd`:

https://github.com/dotnet/aspnetcore/blob/7a0d0de38dbb729b823115d60d2d9ac691d8c292/eng/docker/rhel.Dockerfile#L12

But note we are running the `docker build/run` ourselves in the pipeline. Not using AzDO's docker capability.